lib/commands/backup.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+const Command = require('../command');
+const Promise = require('bluebird');
+const createDebug = require('debug');
+const Zip = require('adm-zip');
+const fs = require('fs');
+const path = require('path');
+const cli = require('../');
+
+const debug = createDebug('ghost-cli:backup');
+
+class BackupCommand extends Command {
+    run(argv) {
+        const instance = this.system.getInstance();
+        const config = instance.config.values;
+        const datetime = (new Date()).toJSON().substring(0, 10);
+
+        let exporter;
+        let saveLocation = process.cwd();
+
+        // @todo check if we need to ensure we're in a ghost directory
```

Comment:
You don't need to check - all commands by default check for a valid install directory unless there's a static "global" property that's set to true. (for example: https://github.com/TryGhost/Ghost-CLI/blob/master/lib/commands/ls.js#L26 - the line disables the valid install check.)

tl;dr you don't need to add a check 😁